db/migrate/20180523230926_create_rents.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rents do |t|
+      t.references :user, foreign_key: true, null: false, default: "user_id"
+      t.references :book, foreign_key: true, null: false, default: "book_id"
+      t.date :from, null: false, default: Date.current()
```

Comment:
Same as before, this will evaluate to the day every migration runs and use it as a default. What do you think about leaving out a default value for the dates?